app/models/manageiq/providers/ibm_power_hmc/infra_manager/event_catcher/stream.rb

```diff
@@ -21,11 +21,10 @@ def poll
     @ems.with_provider_connection do |connection|
       catch(:stop_polling) do
         loop do
-          connection.next_events(false).each do |event|
+          connection.next_events.each do |event|
```

Comment:
Does the argument to `next_events` indicate "blocking" ?  Guessing that is why you don't need the sleep anymore?

Reply:
Just make sure that this worker still shuts down cleanly if no events are raised

Reply:
On the last commit I just reset this file to the last upstream state, I think we will work on a pull request later to fix this issue

Reply:
:+1: sounds good